Lib/fontTools/varLib/featureVars.py

```diff
@@ -414,19 +415,42 @@ def makeSubstitutionsHashable(conditionalSubstitutions):
         condSubst.append((conditionSet, substitutions))
     return condSubst, sorted(allSubstitutions)
 
+class ShifterVisitor(TTVisitor):
+    def __init__(self, shift):
+        self.shift = shift
+
+@ShifterVisitor.register_attrs(
+    (
+        (ot.SubstLookupRecord, ("LookupListIndex")),  # GSUB
+        (ot.PosLookupRecord, ("LookupListIndex")),  # GPOS
```

Comment:
why not add `FeatureRecord` as well here? it has the same "LookupListIndex" attribute

Reply:
`FeatureRecord.LookupListIndex` is a list of numbers, not just a number.

Reply:
I mean, sure, I can add another block. Let me do that.